.github/workflows/buf.yaml

```diff
@@ -0,0 +1,31 @@
+name: buf
+on:
+  push:
+    branches: ["main"]
+  workflow_dispatch: {} # support manual runs
+jobs:
+  push:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: bufbuild/buf-setup-action@v1.9.0
```

Comment:
Should we just pin it to the major `@v1` to ensure this repository always works with the latest. In the odd change it'll allow us to address issues earlier.

Reply:
Dependabot should keep it up to date (and if there is any issue, we should be able to troubleshoot it more easily if a build fails with an upgrade from a known version).